src/awkward/operations/convert.py

```diff
@@ -2846,15 +2850,15 @@ def handle_arrow(obj):
                 return ak.layout.RecordArray(child_array, obj.schema.names)
 
         elif isinstance(obj, pyarrow.lib.Table):
-            chunks = []
```

Comment:
I renamed this var to avoid confusion. `to_batches` creates RecordBatch obj that we concat.